src/components/SearchModal/CurrencyList/CurrencyList.component.tsx

```diff
@@ -149,7 +149,7 @@ export const CurrencyList = ({
   )
 
   const Row = useCallback(
-    ({ data, index, style }) => {
+    ({ data, index, style }: { data: any; index: number; style: CSSProperties }) => {
```

Comment:
Ins't `data` a Curreny array?

Reply:
Will fix these types along with the webpack5 upgrade.